### DISCUSSION TITLE
multi-user (multiple accounts) feature

### DISCUSSION THREAD

**Discussion by mcodenie** [community/community#29933] [Enterprise]
Is there any plan to login a multi-user(multiple accounts)? similar to Teams/Outlook where you click to specific user and able to check. Currently only Enterprise can be added aside from personal account.

**Comment by jfullstackdev** [comment]
it's actually the reverse, check this out 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/merging-multiple-personal-accounts
